### PR TITLE
Focus view updates to the UG

### PR DESCRIPTION
**Pull request by cpandhi**
related issue [#1869](https://github.ibm.com/IBMa/E2E/issues/1869)

### CODE REVIEW

File: accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx [hunk 1]
```diff
@@ -704,7 +709,62 @@ class UsingACApp extends React.Component<{}, UsingACAppState> {
               </ol>
             </div>
 
-            <h3 id="a11y_assess">6.2 Accessibility Assessment</h3>
+            <h3 id="focus_view">6.2 Focus View</h3>
+            <div className="pa">
+              The <strong>'Focus view'</strong> switch allows you to to switch between viewing all issues on the page, or only the issues for a selected element or component in the DOM. To focus on any individual element or component:
```
Comment (eirckr) [inline accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx:714]: switch allows you to to switch. Just one "to"

Comment (cpandhi) [reply accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx]: Done!

**Comment by cpandhi** [comment]
@eirckr  Pls hold this pull request, need to push more updates to the User Guide

File: accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx [hunk 2]
```diff
@@ -704,7 +709,62 @@ class UsingACApp extends React.Component<{}, UsingACAppState> {
...
+                <li>Select the element or the component in the <strong>'DOM view'</strong>, or</li>
+                <li>
+                  <strong>'Right-click'</strong> on a page element and select 'Inspect' (Chrome) or ‘Inspect Element’ (Firefox).
+                </li>
+                <li>
+                  Select the element name in the <strong>‘Focus View’</strong> switch to view only the issues for that element and its children.
+                </li>
+                <li>Select the <strong>'All'</strong> option in the <strong>‘Focus View’</strong> switch to see all issues for the page again.</li>
+              
+              By default, after the first scan of a page, all issues are shown, and the <strong>'html'</strong> element is selected, as shown in this screenshot:
```
Comment (sharitrewin) [inline accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx:727]: html should be &lt;html&gt; and doesn't need quotes or emphasis as we are talking about the actual element in the DOM, not the content of the switch component.

File: accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx [hunk 3]
```diff
@@ -704,7 +709,62 @@ class UsingACApp extends React.Component<{}, UsingACAppState> {
...
+                    />
+              </p>
+              <br/>
+              In this screen shot, the <strong>'search input'</strong> element in the DOM has been selected, and the 
+              <strong>‘Focus View’</strong> switch has has been set to show all the issues including the selected <strong>‘search input’</strong>:
+              <br/>
+              <p>
+                    <img
+                      src="assets/img/searchFocusViewAll.png"
+                      alt="Accessibility Checker screenshot - Focus view of 'search input' will all issues"
```
Comment (sharitrewin) [inline accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx:746]:
```suggestion
                      alt="Accessibility Checker screenshot. Focus view switch options are 'input' and 'All' (selected) and all issues on the page are shown"
```

**Review by sharitrewin — CHANGES_REQUESTED** [review]
I've suggested some edits to the formatting for consistency with the rest of the user guide, and updates to the alt text for the screenshots.

File: accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx (hunk 2 above)
Comment (cpandhi) [reply accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx]: agree, corrected.

File: accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx [hunk 4]
```diff
@@ -548,8 +548,8 @@ class UsingACApp extends React.Component<{}, UsingACAppState> {
                   <p style={{ marginTop: "0rem" }}>
                     To <strong>filter</strong> issues and focus only on
                     violations, items that need review or recommendations,
-                    select the issue type in the issue count region below the
-                    'Scan' button. Select the same issue type again to see all
+                    deselect the checkbox by the issue type in the issue count region below the
+                    'Scan' button to exclude the issue type from the results. Select the checkbox by the issue type to include the issue type in the results.
                     issue types.
```
Comment (sharitrewin) [inline accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx:553]: Does this line (553) 'issue types.' need to be deleted?

File: accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx [hunk 5]
```diff
@@ -688,15 +688,15 @@ class UsingACApp extends React.Component<{}, UsingACAppState> {
...
+                    highlighted and opened.
                   </p>
                 </li>
                 <li>
                   <p style={{ marginTop: "0rem" }}>
-                    Use the <strong>‘Report’</strong> button to download a
+                    Use the <strong>‘Reports’</strong> button to download a
                     standalone HTML report that includes the same three views
-                    and all the help information provided in the checker.
+                    and all the help information provided in the checker and a MS excel spreadsheet report for easy data manipulation.
```
Comment (sharitrewin) [inline accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx:699]:
```suggestion
                    and all the help information provided in the checker and a MS Excel spreadsheet report for easy data manipulation.
```

File: accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx [hunk 6]
```diff
@@ -1121,10 +1121,9 @@ class UsingACApp extends React.Component<{}, UsingACAppState> {
               </ol>
             </div>
 
-            <h2 id="the_report">7. Accessibility Checker report</h2>
+            <h2 id="the_report">7. Accessibility Checker reports</h2>
             <p>
-              The Accessibility Checker provides a full report that you can
-              download as HTML. To open the Accessibility Checker report:
+              The Accessibility Checker provides two kinds of reports to download, one as HTML and one as MS excel spreadsheet. To open the Accessibility Checker reports:
```
Comment (sharitrewin) [inline accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx:1126]:
```suggestion
              The Accessibility Checker provides two kinds of reports to download, one as HTML and one as MS Excel spreadsheet. To open the Accessibility Checker reports:
```

**Review by sharitrewin — CHANGES_REQUESTED** [review]
Looking good.  Excel should be capitalized everywhere and it looks like there is a stray line that needs to be deleted.

File: accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx (hunk 4 above)
Comment (cpandhi) [reply accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx:553]: yes, thanks.

**Review by sharitrewin — DISMISSED** [review]

**Review by tombrunet — APPROVED** [review]

**Pull request merged**